test/fixtures/variables/extra-plugin.js

```diff
@@ -0,0 +1,45 @@
+module.exports = class ExtraPlugin {
```

Comment:
Why not in TS ? This would allow you not to have to ignore this file in `.eslintignore`

Reply:
Good point I did not think about that.

However I think that might bring in more complexity: I would have to include the compiled `*.js` plugin file (not the `*.ts` one). Also I don't really want to bother defining types everywhere: this is a fixture file and it's replicating existing JS code that I copy/pasted from problematic scenarios from other JS plugins so…